code/modules/crafting/recipes.dm

```diff
@@ -323,7 +332,7 @@
 	time = 30
```

Comment:
this is fine. Surprised that it didnt get overwritten, when the lance recipe did.